scripts/qa_crowbarsetup.sh

```diff
@@ -2944,6 +2944,11 @@ function deploy_single_proposal()
         ceph)
             [[ -n "$deployceph" ]] || continue
             ;;
+        magnum)
+            if iscloudver 6plus ; then
```

Comment:
Shouldn't this just skip the magnum deployment depending of the want_magnum setting? (See e.g. want_ceph, want_tempest and so on)